projects/graph/src/graph_demo.py

```diff
@@ -4,13 +4,133 @@
 Demonstration of Graph and BokehGraph functionality.
 """
 
+import random
 from sys import argv
+from draw import BokehGraph
+from graph import Graph, Vertex
+
+def createDefaultGraph():
+    graph = Graph()  # Instantiate your graph
+    # graph.add_vertex('0')
+    # graph.add_vertex('1')
+    # graph.add_vertex('2')
+    # graph.add_vertex('3')
+    # graph.add_vertex('4')
+    # graph.add_vertex('5')
+    # graph.add_vertex('6')
+    # graph.add_edge('0', '1')
+    # graph.add_edge('2', '3')
+    # graph.add_edge('2', '4')
+    # graph.add_edge('3', '5')
+    # graph.add_edge('5', '6')
+
+    graph.add_vertex(5)
+    graph.add_vertex(2)
+    graph.add_vertex(6)
+    graph.add_vertex(1)
+    graph.add_vertex(4)
+    graph.add_vertex(7)
+    graph.add_vertex(3)
+    graph.add_edge(5, 2)
+    graph.add_edge(5, 6)
+    graph.add_edge(2, 1)
+    graph.add_edge(2, 4)
+    graph.add_edge(4, 3)
+    graph.add_edge(6, 7)
+
+    print(graph.dfs_path(5, 7))
+
```

Comment:
This prints correctly in the terminal, but because it is hard coded in, the same numbers appear connected in the same order - they are just in different locations on the graph.